aws-predictions-tfl/src/main/java/com/amplifyframework/predictions/service/TFLiteTextClassificationService.java

```diff
@@ -0,0 +1,332 @@
+    private static final String LABEL_PATH = "text_classification_labels.txt";
+
+    // The maximum length of an input sentence.
+    private static final int SENTENCE_LEN = 256;
+
+    // Percentage multiplier
+    private static final int PERCENT = 100;
+
+    // Simple delimiter to split words.
+    private static final String SIMPLE_SPACE_OR_PUNCTUATION = "[ ,.!?\n]";
+
+    /*
+     * Reserved values in ImdbDataSet dictionary:
+     * dictionary["<PAD>"] = 0      used for padding
+     * dictionary["<START>"] = 1    mark for the start of a sentence
+     * dictionary["<UNKNOWN>"] = 2  mark for unknown words (OOV)
+     */
+    private static final String PAD = "<PAD>";
+    private static final String START = "<START>";
+    private static final String UNKNOWN = "<UNKNOWN>";
+
+    private final Map<String, Integer> dictionary = new HashMap<>();
+    private final List<String> labels = new ArrayList<>();
+
+    private final Context context;
+
+    private Interpreter tflite;
+    private AtomicBoolean loaded;
+
+    /**
+     * Constructs an instance of service to perform text
+     * sentiment interpretation using Tensorflow Lite
+     * interpreter.
+     * @param context the Android context for loading model
+     */
+    TFLiteTextClassificationService(Context context) {
+        this.context = context;
+        this.loaded = new AtomicBoolean(false);
+
+        try {
+            // Try loading assets now if possible
+            loadIfNotLoaded();
+        } catch (PredictionsException exception) {
+            // Ignore if it fails to load during configuration.
+
+            // This may sound weird, but we want the model to be loaded in as soon as possible.
+            // But we don't want to throw an error that needs to be caught at configuration time,
+            // since these models are supposed to be optional.
+        }
+    }
+
+    /**
+     * Gets the associated service key of this service for
+     * identification.
+     * @return the service key
+     */
+    @NonNull
+    String getServiceKey() {
+        return SERVICE_KEY;
+    }
+
+    /**
+     * Classifies text to analyze associated sentiments.
+     * @param text the text to classify
+     * @param onSuccess notified when classification succeeds
+     * @param onError notified when classification fails
+     */
+    void classify(
+            @NonNull String text,
+            @NonNull Consumer<InterpretResult> onSuccess,
+            @NonNull Consumer<PredictionsException> onError
+    ) {
+        final Sentiment sentiment;
+        try {
+            loadIfNotLoaded();
+            sentiment = fetchSentiment(text);
+        } catch (PredictionsException exception) {
+            onError.accept(exception);
+            return;
+        }
+
+        InterpretResult result = InterpretResult.builder()
+                .sentiment(sentiment)
+                .build();
+        onSuccess.accept(result);
```

Comment:
trivial -- when you have such a small amount of code for the payload, you can inline it:
```
onSuccess.accept(InterpretResult.builder()
    .sentiment(sentiment)
    .build());
```

Actually, you could probably even bring that whole thing up into the `try` block:
```
try {
    onSuccess.accept(InterpretResult.builder()
        .sentiment(fetchSentiment(text))
        .build());
} catch (PredictionsException sentimentDetectionFailure) {
    onError.accept(sentimentDetectionFailure);
}
```